dotnet/Templates/Microsoft.MacCatalyst.Templates/maccatalyst/csharp/.template.config/localize/templatestrings.cs.json

```diff
@@ -1,7 +1,7 @@
-{
+﻿{
   "author": "Microsoft",
-  "name": "Aplikace Mac Catalyst",
-  "description": "Projekt pro vytvoření aplikace .NET Mac Catalyst",
-  "symbols/bundleId/description": "Přepíše CFBundleIdentifier v souboru Info.plist.",
-  "symbols/minOSVersion/description": "Přepíše SupportedOSPlatformVersion v souboru projektu."
+  "name": "Mac Catalyst Application",
+  "description": "A project for creating a .NET Mac Catalyst application",
+  "symbols/bundleId/description": "Overrides CFBundleIdentifier in the Info.plist",
+  "symbols/minOSVersion/description": "Overrides SupportedOSPlatformVersion in the project file"
```

Comment:
@tj-devel709 seems like this is reverting the translations?

Reply:
Yeah something is fishy. These templates are not inside the https://github.com/xamarin/xamarin-macios/tree/main/Localize/loc/cs/dotnet/Templates/Microsoft.MacCatalyst.Templates where the lcl files to be translated are located so I am not really sure what is happening here. I think we should get this PR in first that adds the corresponding lcl files: https://github.com/xamarin/xamarin-macios/pull/17442 and close this one since I think this behavior is not wanted